exercises/worker-versioning/solution/src/main/java/pizzaworkflow/PizzaWorker.java

```diff
@@ -0,0 +1,29 @@
+package pizzaworkflow;
+
+import io.temporal.client.WorkflowClient;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+import io.temporal.worker.Worker;
+import io.temporal.worker.WorkerFactory;
+import io.temporal.worker.WorkerOptions;
+
+public class PizzaWorker {
+  public static void main(String[] args) {
+
+    WorkflowServiceStubs service = WorkflowServiceStubs.newLocalServiceStubs();
+    WorkflowClient client = WorkflowClient.newInstance(service);
+    WorkerFactory factory = WorkerFactory.newInstance(client);
+
+    WorkerOptions workerOptions = WorkerOptions.newBuilder()
+         .setBuildId("v2.1")
```

Comment:
shouldn't this be v2?

Reply:
The solution is what would be there at the final stage, and Part C has a 2.1 being added. I will actually add a comment here for 2.0 as well.